abstractions/model_free/sac/model.py

```diff
@@ -37,23 +44,28 @@ def __init__(self, num_inputs, action_space, device, args):
                 self.log_alpha = torch.zeros(1, requires_grad=True, device=self.device)
                 self.alpha_optim = torch.optim.Adam([self.log_alpha], lr=args.lr)
 
-            self.policy = GaussianPolicy(num_inputs,
+            self.policy = GaussianPolicy(input_shape,
                                          action_space.shape[0],
                                          args.hidden_size,
+                                         args.model_type,
+                                         args.num_frames,
                                          action_space).to(self.device)
             self.policy_optim = torch.optim.Adam(self.policy.parameters(), lr=args.lr)
 
         else:
             self.alpha = 0
             self.automatic_entropy_tuning = False
-            self.policy = DeterministicPolicy(num_inputs,
+            self.policy = DeterministicPolicy(input_shape,
                                               action_space.shape[0],
                                               args.hidden_size,
+                                              args.model_type,
+                                              args.num_frames,
                                               action_space).to(self.device)
             self.policy_optim = torch.optim.Adam(self.policy.parameters(), lr=args.lr)
 
-    def select_action(self, state, evaluate=False):
-        state = torch.FloatTensor(state).to(self.device).unsqueeze(0)
+    def act(self, state, evaluate=False):
+        state = torch.FloatTensor(state).to(self.device)
```

Comment:
`state = torch.from_numpy(state).float().to(self.device)` should be faster

Reply:
Yeah I'll change that.